File: ruby/lib/user_interface/help_command.rb
module UserInterface

	class HelpCommand

		def run(cmd)
			puts "Customer actions:"
			puts "  a sku [num]             Add SKU to basket; num defaults to 1"
			puts "  b                       List basket contents"
			puts "  c                       Checkout"
			puts "  d sku                   Show detailed product info for SKU"
			puts "  p                       List all products"
			puts
			puts "Warehouse actions:"
			puts "  r aisle loc sku num     Replenish SKU with num additional units"
			puts "  s                       Show stock levels"
			puts
			puts "General:"
			puts "  h                       Show this help"
			puts "  q                       Quit"
		end

	end
end

File: python/user_interface/help_command.py
class HelpCommand:

    def run(self, cmd):
        print "Customer actions:"
        print "  a sku [num]             Add SKU to basket; num defaults to 1"
        print "  b                       List basket contents"
        print "  c                       Checkout"
        print "  d sku                   Show detailed product info for SKU"
        print "  p                       List all products"
        print
        print "Warehouse actions:"
        print "  r aisle loc sku num     Replenish SKU with num additional units"
        print "  s                       Show stock levels"
        print
        print "General:"
        print "  h                       Show this help"
        print "  q                       Quit"

